packages/palette/tsconfig.json

```diff
@@ -7,5 +7,6 @@
   },
   "include": [
     "src/**/*"
-  ]
+  ],
+  "references": []
```

Comment:
What is this for?

Reply:
I think this was done by depchek. will remove